### PR TITLE
feat: IST-2429 - Add quickstart docs for both single and multicam

### PR DESCRIPTION
**Pull request by deechauhan**

### CODE REVIEW

**Comment by benmoveai** [comment]
@deechauhan It's probably worthwhile getting Ross to read through it to make sure he's happy with it all.

A few general points:

* One thing that we could sneak into this PR is removing the reference to a `.move` file on [this](https://move-ai.github.io/move-ugc-python/1.9.0/getting-started/usage/take/) page. Ross wanted us to remove any reference to these files as users cannot generate them - and I hadn't realised that there was a reference here
* The `jobs` page ([here](https://move-ai.github.io/move-ugc-python/1.9.0/getting-started/usage/job/)) still has reference to `jobs.create` which is being deprecated. Can this be updated to point to the specific create singlecam/create multicam jobs docs in the quickstart (I think the quickstart makes most sense)
* Take creation docs ([here](https://move-ai.github.io/move-ugc-python/1.9.0/getting-started/usage/take/)) reference only create_singlecam. Should this also reference create_multicam?

**Comment by deechauhan** [comment]
> @deechauhan It's probably worthwhile getting Ross to read through it to make sure he's happy with it all.
> 
> A few general points:
> 
>     * One thing that we could sneak into this PR is removing the reference to a `.move` file on [this](https://move-ai.github.io/move-ugc-python/1.9.0/getting-started/usage/take/) page. Ross wanted us to remove any reference to these files as users cannot generate them - and I hadn't realised that there was a reference here
> 
>     * The `jobs` page ([here](https://move-ai.github.io/move-ugc-python/1.9.0/getting-started/usage/job/)) still has reference to `jobs.create` which is being deprecated. Can this be updated to point to the specific create singlecam/create multicam jobs docs in the quickstart (I think the quickstart makes most sense)
> 
>     * Take creation docs ([here](https://move-ai.github.io/move-ugc-python/1.9.0/getting-started/usage/take/)) reference only create_singlecam. Should this also reference create_multicam?

I've updated the docs for #1,2, for updating the take usage docs, I've included examples and links to take api reference for multicam in the quickstart docs, do you think we need to add it somewhere else as well?

File: docs/getting-started/usage/take.md [hunk 1]
```diff
@@ -34,13 +31,9 @@ take = ugc.takes.create_singlecam(
             device_label="my-device",
             file_id="file-457e23c2-6afc-4913-91f6-36522245d57d",
             format="mp4",
-        ),
```
Comment (benmoveai) [inline docs/getting-started/usage/take.md:37]: If you expand this section you can see that line 24 still talks about the move file. I think we can probably get rid of this entire section and just leave lines 8-22

Comment (deechauhan) [reply docs/getting-started/usage/take.md:37]: I've just removed this section

**Review by benmoveai — APPROVED** [review]

**Review by benmoveai — APPROVED** [review]

**Pull request merged**